examples/elm-pb/elm_pb.cxx

```diff
@@ -350,7 +351,8 @@ class ELMpb : public PhysicsModel {
     //////////////////////////////////////////////////////////////
     auto& globalOptions = Options::root();
     auto& options = globalOptions["highbeta"];
-
+    laplace_perp = options["laplace_perp"].withDefault(false);    
+    // Use Laplace_perp rather than Delp2
```

Comment:
```suggestion
    // Use Laplace_perp rather than Delp2
    laplace_perp = options["laplace_perp"].withDefault(false);    
```
Please put comments above the code.

Why did you add this?

In general, I think we should keep the examples as clean as reasonable possible ...

If this is generally useful, should we switch to only `Laplace_perp`?
If we want to keep this, we could add a wrapper function, that contains the  `if`/`else` code, and then we always call the member function, and keep the code clean.